dot/state/epoch.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/blocktree"
 	"github.com/ChainSafe/gossamer/lib/common"
+	trieproof "github.com/ChainSafe/gossamer/lib/trie/proof"
```

Comment:
nit why not just without alias? 🤔

Reply:
I would rather have trieproof than proof